WordPress/Classes/Utility/WebKitViewController.swift

```diff
@@ -82,13 +82,6 @@ class WebKitViewController: UIViewController {
         fatalError("init(coder:) has not been implemented")
     }
 
-    deinit {
```

Comment:
Sorry if I'm mistaken here, but isn't this KVO? Are KVO observers also removed implicitly?

Reply:
These are indeed KVO. I caught this after creating the PR, but neglected to push a change that restores it.